packages/react-components/src/components/VideoGallery/utils/reactionUtils.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+/** @private */
+export const REACTION_START_DISPLAY_SIZE = 44;
+/* @conditional-compile-remove(reaction) */
+/** @private */
+export const REACTION_NUMBER_OF_ANIMATION_FRAMES = 51;
+/* @conditional-compile-remove(reaction) */
+/** @private */
+export const REACTION_SCREEN_SHARE_ANIMATION_TIME_MS = 4133;
+
+/* @conditional-compile-remove(reaction) */
+/** @private */
+export const getCombinedKey = (userId: string, reactionType: string, receivedAt: Date): string => {
+  const receivedTime =
```

Comment:
Instead of manually extracting each time unit, can use `toISOString` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

```ts
  const receivedTime = receivedAt.toISOString();
```